application/models/Tracker/Sites/ReadMangaToday.php

```diff
@@ -0,0 +1,46 @@
+<?php declare(strict_types=1); defined('BASEPATH') OR exit('No direct script access allowed');
+
+class ReadMangaToday extends Base_Site_Model {
+	public $titleFormat   = '/^[a-zA-Z0-9_-]+$/';
+	public $chapterFormat = '/^[0-9\.]+$/';
+
+	public function getFullTitleURL(string $title_url) : string {
+		return "http://www.readmanga.today/{$title_url}";
+	}
+
+	public function getChapterData(string $title_url, string $chapter) : array {
+		//http://www.readmanga.today/saiki-kusuo-no-psi-nan/128
+		return [
+			'url'    => $this->getFullTitleURL($title_url).$chapter.'/',
```

Comment:
MangaCow urls all have trailing slashes by default, while readmanga.today does not.